zerver/data_import/slack.py

```diff
@@ -269,6 +292,14 @@ def users_to_zerver_userprofile(
 
         logging.info("%s -> %s", user["name"], userprofile_dict["email"])
 
+    bot_suffix = BotSuffix(all_bot_emails)
+
+    for user in zerver_userprofile:
+        suffix = bot_suffix.get_suffix(user["email"])
+        user["email"] = user["email"].split("@")
+        user["email"][0] += suffix
+        user["email"] = "@".join(user["email"])
```

Comment:
Let's use different variable names for the different concepts, e.g. 
`email_prefix, email_suffix = user["email"].split("@")`

Reply:
Sure.